pkg/test/xds/client/go.mod

```diff
@@ -3,20 +3,20 @@ module github.com/Kong/kuma/pkg/test/xds/client
 go 1.16
 
 require (
-	github.com/envoyproxy/go-control-plane v0.9.7
-	github.com/golang/protobuf v1.4.2
+	github.com/envoyproxy/go-control-plane v0.9.9-0.20210217033140-668b12f5399d
```

Comment:
why do we need this upgrade?

Reply:
This was modified as part of the rebuilding protobufs make process. I can revert and test that it still works.